components/brave_rewards/resources/extension/brave_rewards/BUILD.gn

```diff
@@ -34,7 +35,8 @@ transpile_web_ui("brave_rewards_panel") {
 
   entry_points = [
     ["brave_rewards_panel", rebase_path("brave_rewards_panel.tsx")],
-    ["brave_rewards_panel_background", rebase_path("background.ts")]
+    ["brave_rewards_panel_background", rebase_path("background.ts")],
+    ["brave_rewards_panel_content", rebase_path("twitter.ts")]
```

Comment:
would be ideal to name the output something twitter-specific (e.g. `brave_rewards_panel_content_twitter`) and put twitter.ts in a directory that is explicit about the content-script context (e.g. `content-scriptps/twitter`)

Reply:
Good idea, will get that in there!